testament/specs.nim

```diff
@@ -265,6 +273,10 @@ proc isCurrentBatch*(testamentData: TestamentData; filename: string): bool =
 
 proc parseSpec*(filename: string): TSpec =
   result.file = filename
+  if retryContainer.retry:
+    if retryContainer.names.anyIt(it in result.file):
+      result.err = reDisabled
```

Comment:
You're not returning early or anything, so it's still doing the rest of the work.